core/init_test.go

```diff
@@ -7,6 +7,9 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// TODO: We need more test coverage for Init and IsInit part
```

Comment:
Someone should always be attributed to the TODO :D so maybe either you or @Wondertan

Reply:
According to Ismail, the person who created the TODO should themself.